Frameworks/Foundation/NSSet.mm

```diff
@@ -554,12 +573,93 @@ - (void)setValue:(id)value forKey:(NSString*)key {
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
 */
 - (NSString*)descriptionWithLocale:(id)locale {
-    UNIMPLEMENTED();
-    return StubReturn();
+    return [self descriptionWithLocale:locale indent:0];
+}
+
+/**
+ @Status Interoperable
+*/
+- (NSString*)description {
+    return [self descriptionWithLocale:nil indent:0];
+}
+
+- (NSString*)descriptionWithLocale:(id)locale indent:(NSUInteger)level {
```

Comment:
There's no real way to keep NSArray's description interoperable while supporting this, and I don't think the gains of cutting down code duplication would be worth tying these together.  I'm flexible on this if we think it's the right move.